wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -14,11 +14,11 @@
 							<h4>Allow users to express a broader range of empathy with the Engaging Buttons plugin.</h4>
 						</div>
 						<p>It’s easy to click “Like” on a heartwarming story about a local hero. But what about a fair, but counter-attitudinal, post in a comment section? That may make it a little more challenging to “Like.”</p>
-						<p>Through our research on <a href="http://mediaengagement.org/research/social-media-buttons/">engagement buttons</a>, we found that <strong>people were more likely to click “Respect” over “Like”</strong> in comment sections, and significantly more likely to click “Respect” when the view expressed differed from their own.</p>
+						<p>Through our research on <a href="{{site.url}}/research/social-media-buttons/">engagement buttons</a>, we found that <strong>people were more likely to click “Respect” over “Like”</strong> in comment sections, and significantly more likely to click “Respect” when the view expressed differed from their own.</p>
 						<p><a class="btn btn-primary" href="https://wordpress.org/plugins/engaging-buttons/">Download Plugin</a> <a class="btn btn-default" href="https://github.com/engagingnewsproject/engaging-button/">View Source</a></p>
 					</div>
 					<div class="col-md-6">
```

Comment:
It doesn't look like `col-md-6` does anything. I believe it was previously so we could float the image on the right/left. Could you apply some of the same classes or CSS from the homepage layout to this section to get it a little snazzier? Also, I think there's no page padding on small screens.